package.json

```diff
@@ -11,6 +11,7 @@
     "prettier": "prettier './**/*.js' './**/*.sol' './**/*.md' './**/*.ts'",
     "test": "lerna run --stream --concurrency=4 test",
     "test-concurrent": "lerna run --stream --concurrency=4 test --ignore @uma/serverless-orchestration --ignore @uma/affiliates && lerna run --stream --concurrency=1 test --scope @uma/serverless-orchestration --scope @uma/affiliates",
+    "test-concurrent-test": "lerna run --stream --concurrency=4 test --scope ${PACKAGES_CHANGES}",
```

Comment:
As discussed today, I think you could just use `yarn test --scope ${SOME_PACKAGE_NAME}`